web/client/codechecker_client/cmd/cmd.py

```diff
@@ -994,7 +992,7 @@ def __register_runs(parser):
                             "midnight (00:00:00) is used).")
 
     # Get available sort types.
-    sort_type_values = ttypes.RunSortType._NAMES_TO_VALUES.values()
+    sort_type_values = list(ttypes.RunSortType._NAMES_TO_VALUES.values())
```

Comment:
Some redundant `list`s here as well.

Reply:
This is not redundant a list is needed below:
```
    # Set 'date' as a default sort type.
    default_sort_type = sort_types[
        sort_type_values.index(ttypes.RunSortType.DATE)]
```